cishouseholds/pipeline/pipeline_stages.py

```diff
@@ -123,7 +123,7 @@ def csv_to_table(file_operations: list):
             schema,
             column_map,
             file["drop_not_found"],
-        )
+        ).distinct()
```

Comment:
We don't want to do this, because it will mask any duplicates before the validation that you've added below